### PR TITLE
[docs] Update ansible batch nodes

### PR DESCRIPTION
**Pull request by kirill-kashin**
## Description
Update dockumentation for add a batch of static nodes to a cluster

## Why do we need it, and what problem does it solve?
Clearer description of documentation

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: docs
summary: Minor updates.
type: chore
impact_level: low
```

### CODE REVIEW

File: modules/040-node-manager/docs/FAQ_RU.md [hunk 1]
```diff
@@ -60,7 +60,7 @@ search: добавить ноду в кластер, добавить узел 
 
    Сохраните полученный токен, и добавьте в поле `token:` playbook'а Ansible на дальнейших шагах.
 
-1. Получите Kubernetes API-токен для специального `ServiceAccount`, которым управляет Deckhouse:
+1. Если версия меньше 1.25 Получите Kubernetes API-токен для специального `ServiceAccount`, которым управляет Deckhouse:
```
Comment (z9r5) [inline modules/040-node-manager/docs/FAQ_RU.md:63]:
```suggestion
1. Если версия Kubernetes меньше 1.25, получите Kubernetes API-токен для специального ServiceAccount'а, которым управляет Deckhouse:
```

File: modules/040-node-manager/docs/FAQ.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ You can automate the bootstrap process with any automation platform you prefer.
 
    Save the token you got and add it to the `token:` field of the Ansible playbook in the next steps.
 
-1. Get Kubernetes API token for special `ServiceAccount` that Deckhouse manages:
+1. If the version is smaller 1.25 Get Kubernetes API token for special `ServiceAccount` that Deckhouse manages:
```
Comment (z9r5) [inline modules/040-node-manager/docs/FAQ.md:63]:
```suggestion
1. If the Kubernetes version is smaller than 1.25, get a Kubernetes API token for a special ServiceAccount that Deckhouse manages:
```

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6259](https://github.com/deckhouse/deckhouse/pull/6259) to the branch [release-1.53](https://github.com/deckhouse/deckhouse/tree/release-1.53) successful!

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [6260](https://github.com/deckhouse/deckhouse/pull/6260) to the branch [release-1.52](https://github.com/deckhouse/deckhouse/tree/release-1.52) successful!

**Pull request merged**